arthur_bench/models/scoring.py

```diff
@@ -6,15 +6,22 @@ class HallucinationScoreRequest(BaseModel):
     Request for hallucination classification
     """
 
-    response: str
+    user_input: str
     """
-    Model generated response
+    User input with which to determine if the model generated response is supported
```

Comment:
General question on this API: is it valuable to distinguish between `user_input` and `context` here? or does it make more sense to keep the concept of "context" more general as any conversation or world context, with any client of the hallucination API choosing how to build the context?

in the bench case, we would concatenated input and context